plugins/DuskVerb/tools/tuner/vintage_minisweep3.py

```
@@ -0,0 +1,103 @@
+            [sys.executable, str(REPO / "plugins/DuskVerb/tools/tuner/full_check.py"),
+             str(out_dir), str(ANCHOR), "--name", "BH", "--category", "Halls"],
+            capture_output=True, text=True, timeout=60)
+    except subprocess.TimeoutExpired:
+        return (999, idx, m, t, hc, gt, lc, out_dir)
+    fails = 999
+    for line in r.stdout.splitlines():
+        if "GATE(S) FAILED" in line:
+            try:
+                fails = int(line.split("✗")[1].strip().split()[0])
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)